<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Biological Sequences Retrieval and Analysis</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Biological Sequences Retrieval and Analysis
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;seqinr&rsquo; version 4.2-5</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="# "> </a>
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#I">I</a>
<a href="#J">J</a>
<a href="#K">K</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#O">O</a>
<a href="#P">P</a>
<a href="#Q">Q</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#U">U</a>
<a href="#W">W</a>
<a href="#Z">Z</a>
</p>

<table width="100%">
<tr><td style="width: 25%;"><a href="seqinr-package.html">seqinr-package</a></td>
<td>Biological Sequences Retrieval and Analysis</td></tr>
</table>

<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="a.html">a</a></td>
<td>Converts amino-acid three-letter code into the one-letter one</td></tr>
<tr><td style="width: 25%;"><a href="aaa.html">aaa</a></td>
<td>Converts amino-acid one-letter code into the three-letter one</td></tr>
<tr><td style="width: 25%;"><a href="aacost.html">aacost</a></td>
<td>Aerobic cost of amino-acids in Escherichia coli and G+C classes</td></tr>
<tr><td style="width: 25%;"><a href="aaindex.html">aaindex</a></td>
<td>List of 544 physicochemical and biological properties for the 20 amino-acids</td></tr>
<tr><td style="width: 25%;"><a href="AAstat.html">AAstat</a></td>
<td>To Get Some Protein Statistics</td></tr>
<tr><td style="width: 25%;"><a href="acnucopen.html">acnucclose</a></td>
<td>open and close a remote access to an ACNUC database</td></tr>
<tr><td style="width: 25%;"><a href="acnucopen.html">acnucopen</a></td>
<td>open and close a remote access to an ACNUC database</td></tr>
<tr><td style="width: 25%;"><a href="al2bp.html">al2bp</a></td>
<td>To Convert a forensic microsatellite allele name into its length in base pairs</td></tr>
<tr><td style="width: 25%;"><a href="alllistranks.html">alllistranks</a></td>
<td>To get the count of existing lists and all their ranks on server</td></tr>
<tr><td style="width: 25%;"><a href="alllistranks.html">alr</a></td>
<td>To get the count of existing lists and all their ranks on server</td></tr>
<tr><td style="width: 25%;"><a href="amb.html">amb</a></td>
<td>Expansion of IUPAC nucleotide symbols</td></tr>
<tr><td style="width: 25%;"><a href="AnoukResult.html">AnoukResult</a></td>
<td>Expected numeric results for Ka and Ks computation</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.alignment</a></td>
<td>Constructor for class alignment</td></tr>
<tr><td style="width: 25%;"><a href="as.matrix.alignment.html">as.matrix.alignment</a></td>
<td>as.matrix.alignment</td></tr>
<tr><td style="width: 25%;"><a href="SeqAcnucWeb.html">as.SeqAcnucWeb</a></td>
<td>Sequence coming from a remote ACNUC data base</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastaAA.html">as.SeqFastaAA</a></td>
<td>AA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastadna.html">as.SeqFastadna</a></td>
<td>Class for DNA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="SeqFrag.html">as.SeqFrag</a></td>
<td>Class for sub-sequences</td></tr>
<tr><td style="width: 25%;"><a href="autosocket.html">autosocket</a></td>
<td>Returns a socket to the last opened database</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="baselineabif.html">baselineabif</a></td>
<td>Estimation of baseline value</td></tr>
<tr><td style="width: 25%;"><a href="bma.html">bma</a></td>
<td>Computing an IUPAC nucleotide symbol</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="c2s.html">c2s</a></td>
<td>conversion of a vector of chars into a string</td></tr>
<tr><td style="width: 25%;"><a href="cai.html">cai</a></td>
<td>Codon Adaptation Index</td></tr>
<tr><td style="width: 25%;"><a href="caitab.html">caitab</a></td>
<td>Codon Adaptation Index (CAI) w tables</td></tr>
<tr><td style="width: 25%;"><a href="countfreelists.html">cfl</a></td>
<td>The number of free lists available and annotation lines in an ACNUC server</td></tr>
<tr><td style="width: 25%;"><a href="chargaff.html">chargaff</a></td>
<td>Base composition in ssDNA for 7 bacterial DNA</td></tr>
<tr><td style="width: 25%;"><a href="choosebank.html">choosebank</a></td>
<td>To select a database structured under ACNUC and located on the web</td></tr>
<tr><td style="width: 25%;"><a href="circle.html">circle</a></td>
<td>Draws a circle</td></tr>
<tr><td style="width: 25%;"><a href="crelistfromclientdata.html">clfcd</a></td>
<td>To create on server an ACNUC list from data lines sent by client</td></tr>
<tr><td style="width: 25%;"><a href="acnucopen.html">clientid</a></td>
<td>open and close a remote access to an ACNUC database</td></tr>
<tr><td style="width: 25%;"><a href="closebank.html">closebank</a></td>
<td>To close a remote ACNUC database</td></tr>
<tr><td style="width: 25%;"><a href="clustal.html">clustal</a></td>
<td>Example of results obtained after a call to read.alignment</td></tr>
<tr><td style="width: 25%;"><a href="col2alpha.html">col2alpha</a></td>
<td>To use a standard color with an alpha transparency chanel</td></tr>
<tr><td style="width: 25%;"><a href="comp.html">comp</a></td>
<td>complements a nucleic acid sequence</td></tr>
<tr><td style="width: 25%;"><a href="computePI.html">computePI</a></td>
<td>To Compute the Theoretical Isoelectric Point</td></tr>
<tr><td style="width: 25%;"><a href="consensus.html">con</a></td>
<td>Consensus and profiles for sequence alignments</td></tr>
<tr><td style="width: 25%;"><a href="consensus.html">consensus</a></td>
<td>Consensus and profiles for sequence alignments</td></tr>
<tr><td style="width: 25%;"><a href="count.html">count</a></td>
<td>Composition of dimer/trimer/etc oligomers</td></tr>
<tr><td style="width: 25%;"><a href="countfreelists.html">countfreelists</a></td>
<td>The number of free lists available and annotation lines in an ACNUC server</td></tr>
<tr><td style="width: 25%;"><a href="countsubseqs.html">countsubseqs</a></td>
<td>Number of subsequences in an ACNUC list</td></tr>
<tr><td style="width: 25%;"><a href="crelistfromclientdata.html">crelistfromclientdata</a></td>
<td>To create on server an ACNUC list from data lines sent by client</td></tr>
<tr><td style="width: 25%;"><a href="countsubseqs.html">css</a></td>
<td>Number of subsequences in an ACNUC list</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="dia.bactgensize.html">dia.bactgensize</a></td>
<td>Distribution of bacterial genome size from GOLD</td></tr>
<tr><td style="width: 25%;"><a href="get.db.growth.html">dia.db.growth</a></td>
<td>Get the exponential growth of nucleic acid database content</td></tr>
<tr><td style="width: 25%;"><a href="dinucl.html">dinucl</a></td>
<td>Mean zscore on 242 complete bacterial chromosomes</td></tr>
<tr><td style="width: 25%;"><a href="dist.alignment.html">dist.alignment</a></td>
<td>Pairwise Distances from Aligned Protein or DNA/RNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="dotchart.uco.html">dotchart.uco</a></td>
<td>Cleveland plot for codon usage tables</td></tr>
<tr><td style="width: 25%;"><a href="dotPlot.html">dotPlot</a></td>
<td>Dot Plot Comparison of two sequences</td></tr>
<tr><td style="width: 25%;"><a href="draw.oriloc.html">draw.oriloc</a></td>
<td>Graphical representation for nucleotide skews in prokaryotic chromosomes.</td></tr>
<tr><td style="width: 25%;"><a href="draw.rearranged.oriloc.html">draw.rearranged.oriloc</a></td>
<td>Graphical representation for rearranged nucleotide skews in prokaryotic chromosomes.</td></tr>
<tr><td style="width: 25%;"><a href="draw.recstat.html">draw.recstat</a></td>
<td>Graphical representation of a recstat analysis.</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="ec999.html">ec999</a></td>
<td>999 coding sequences from E. coli</td></tr>
<tr><td style="width: 25%;"><a href="ECH.html">ECH</a></td>
<td>Forensic Genetic Profile Allelic Ladder Raw Data</td></tr>
<tr><td style="width: 25%;"><a href="EXP.html">EXP</a></td>
<td>Vectors of coefficients to compute linear forms.</td></tr>
<tr><td style="width: 25%;"><a href="extractseqs.html">exseq</a></td>
<td>To extract the sequences information of a sequence or a list of sequence in different formats</td></tr>
<tr><td style="width: 25%;"><a href="extract.breakpoints.html">extract.breakpoints</a></td>
<td>Extraction of breakpoint positions on the rearranged nucleotide skews.</td></tr>
<tr><td style="width: 25%;"><a href="extractseqs.html">extractseqs</a></td>
<td>To extract the sequences information of a sequence or a list of sequence in different formats</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="read.fasta.html">FASTA</a></td>
<td>read FASTA formatted files</td></tr>
<tr><td style="width: 25%;"><a href="fasta.html">fasta</a></td>
<td>Example of results obtained after a call to read.alignment</td></tr>
<tr><td style="width: 25%;"><a href="fastacc.html">fastacc</a></td>
<td>Fast Allele in Common Count</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="gb2fasta.html">gb2fasta</a></td>
<td>Conversion of GenBank file into fasta file</td></tr>
<tr><td style="width: 25%;"><a href="gbk2g2.html">gbk2g2</a></td>
<td>Conversion of a GenBank format file into a glimmer-like one</td></tr>
<tr><td style="width: 25%;"><a href="gbk2g2.euk.html">gbk2g2.euk</a></td>
<td>Conversion of a GenBank format file into a glimmer-like one. Eukaryotic version.</td></tr>
<tr><td style="width: 25%;"><a href="GC.html">GC</a></td>
<td>Calculates the fractional G+C content of nucleic acid sequences.</td></tr>
<tr><td style="width: 25%;"><a href="GC.html">GC1</a></td>
<td>Calculates the fractional G+C content of nucleic acid sequences.</td></tr>
<tr><td style="width: 25%;"><a href="GC.html">GC2</a></td>
<td>Calculates the fractional G+C content of nucleic acid sequences.</td></tr>
<tr><td style="width: 25%;"><a href="GC.html">GC3</a></td>
<td>Calculates the fractional G+C content of nucleic acid sequences.</td></tr>
<tr><td style="width: 25%;"><a href="gcO2.html">gcO2</a></td>
<td>GC content and aerobiosis in bacteria</td></tr>
<tr><td style="width: 25%;"><a href="GC.html">GCpos</a></td>
<td>Calculates the fractional G+C content of nucleic acid sequences.</td></tr>
<tr><td style="width: 25%;"><a href="gcT.html">gcT</a></td>
<td>GC content and temperature in bacteria</td></tr>
<tr><td style="width: 25%;"><a href="get.db.growth.html">get.db.growth</a></td>
<td>Get the exponential growth of nucleic acid database content</td></tr>
<tr><td style="width: 25%;"><a href="get.ncbi.html">get.ncbi</a></td>
<td>Bacterial complete genome data from ncbi ftp site</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot.default</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot.list</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot.logical</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot.qaw</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot.SeqAcnucWeb</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot.SeqFastaAA</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">getAnnot.SeqFastadna</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="isenum.html">getAttributsocket</a></td>
<td>Get the ACNUC number of a sequence from its name or accession number</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.character</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.default</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.list</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.logical</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.qaw</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.SeqAcnucWeb</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.SeqFastaAA</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.SeqFastadna</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getFrag.html">getFrag.SeqFrag</a></td>
<td>Generic function to extract sequence fragments</td></tr>
<tr><td style="width: 25%;"><a href="getKeyword.html">getKeyword</a></td>
<td>Generic function to get keywords associated to sequences</td></tr>
<tr><td style="width: 25%;"><a href="getKeyword.html">getKeyword.default</a></td>
<td>Generic function to get keywords associated to sequences</td></tr>
<tr><td style="width: 25%;"><a href="getKeyword.html">getKeyword.list</a></td>
<td>Generic function to get keywords associated to sequences</td></tr>
<tr><td style="width: 25%;"><a href="getKeyword.html">getKeyword.logical</a></td>
<td>Generic function to get keywords associated to sequences</td></tr>
<tr><td style="width: 25%;"><a href="getKeyword.html">getKeyword.qaw</a></td>
<td>Generic function to get keywords associated to sequences</td></tr>
<tr><td style="width: 25%;"><a href="getKeyword.html">getKeyword.SeqAcnucWeb</a></td>
<td>Generic function to get keywords associated to sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.character</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.default</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.list</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.logical</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.qaw</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.SeqAcnucWeb</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.SeqFastaAA</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.SeqFastadna</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getLength.html">getLength.SeqFrag</a></td>
<td>Generic function to get the length of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getlistrank.html">getlistrank</a></td>
<td>To get the rank of a list from its name</td></tr>
<tr><td style="width: 25%;"><a href="getliststate.html">getliststate</a></td>
<td>Asks for information about an ACNUC list of specified rank</td></tr>
<tr><td style="width: 25%;"><a href="getLocation.html">getLocation</a></td>
<td>Generic function to get the location of subsequences on the parent sequence</td></tr>
<tr><td style="width: 25%;"><a href="getLocation.html">getLocation.default</a></td>
<td>Generic function to get the location of subsequences on the parent sequence</td></tr>
<tr><td style="width: 25%;"><a href="getLocation.html">getLocation.list</a></td>
<td>Generic function to get the location of subsequences on the parent sequence</td></tr>
<tr><td style="width: 25%;"><a href="getLocation.html">getLocation.logical</a></td>
<td>Generic function to get the location of subsequences on the parent sequence</td></tr>
<tr><td style="width: 25%;"><a href="getLocation.html">getLocation.qaw</a></td>
<td>Generic function to get the location of subsequences on the parent sequence</td></tr>
<tr><td style="width: 25%;"><a href="getLocation.html">getLocation.SeqAcnucWeb</a></td>
<td>Generic function to get the location of subsequences on the parent sequence</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.default</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.list</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.logical</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.qaw</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.SeqAcnucWeb</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.SeqFastaAA</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.SeqFastadna</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="getName.html">getName.SeqFrag</a></td>
<td>Generic function to get the names of sequences</td></tr>
<tr><td style="width: 25%;"><a href="isenum.html">getNumber.socket</a></td>
<td>Get the ACNUC number of a sequence from its name or accession number</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.character</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.default</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.list</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.logical</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.qaw</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.SeqAcnucWeb</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.SeqFastaAA</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.SeqFastadna</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getSequence.html">getSequence.SeqFrag</a></td>
<td>Generic function to get sequence data</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.character</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.default</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.list</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.logical</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.qaw</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.SeqAcnucWeb</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.SeqFastadna</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getTrans.html">getTrans.SeqFrag</a></td>
<td>Generic function to translate coding sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="getType.html">getType</a></td>
<td>To get available subsequence types in an opened ACNUC database</td></tr>
<tr><td style="width: 25%;"><a href="gfrag.html">gfrag</a></td>
<td>Extract sequence identified by name or by number from an ACNUC server</td></tr>
<tr><td style="width: 25%;"><a href="ghelp.html">ghelp</a></td>
<td>Get help from an ACNUC server</td></tr>
<tr><td style="width: 25%;"><a href="getliststate.html">gln</a></td>
<td>Asks for information about an ACNUC list of specified rank</td></tr>
<tr><td style="width: 25%;"><a href="getlistrank.html">glr</a></td>
<td>To get the rank of a list from its name</td></tr>
<tr><td style="width: 25%;"><a href="getliststate.html">gls</a></td>
<td>Asks for information about an ACNUC list of specified rank</td></tr>
<tr><td style="width: 25%;"><a href="gs500liz.html">gs500liz</a></td>
<td>GS500LIZ size standards</td></tr>
</table>

<h2><a name="I">-- I --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="identifiler.html">identifiler</a></td>
<td>Identifiler allele names</td></tr>
<tr><td style="width: 25%;"><a href="SeqAcnucWeb.html">is.SeqAcnucWeb</a></td>
<td>Sequence coming from a remote ACNUC data base</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastaAA.html">is.SeqFastaAA</a></td>
<td>AA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastadna.html">is.SeqFastadna</a></td>
<td>Class for DNA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="SeqFrag.html">is.SeqFrag</a></td>
<td>Class for sub-sequences</td></tr>
<tr><td style="width: 25%;"><a href="isenum.html">isenum</a></td>
<td>Get the ACNUC number of a sequence from its name or accession number</td></tr>
<tr><td style="width: 25%;"><a href="isenum.html">isn</a></td>
<td>Get the ACNUC number of a sequence from its name or accession number</td></tr>
</table>

<h2><a name="J">-- J --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="JLO.html">JLO</a></td>
<td>Forensic Genetic Profile Raw Data</td></tr>
</table>

<h2><a name="K">-- K --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="kaks.html">kaks</a></td>
<td>Ka and Ks, also known as dn and ds, computation</td></tr>
<tr><td style="width: 25%;"><a href="kaks-torture.html">kaksTorture</a></td>
<td>Expected numeric results for Ka and Ks in extreme cases</td></tr>
<tr><td style="width: 25%;"><a href="knowndbs.html">kdb</a></td>
<td>Description of databases known by an ACNUC server</td></tr>
<tr><td style="width: 25%;"><a href="knowndbs.html">knowndbs</a></td>
<td>Description of databases known by an ACNUC server</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="lseqinr.html">lseqinr</a></td>
<td>To see what's inside the package seqinr</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="m16j.html">m16j</a></td>
<td>Fragment of the E. coli chromosome</td></tr>
<tr><td style="width: 25%;"><a href="mase.html">mase</a></td>
<td>Example of results obtained after a call to read.alignment</td></tr>
<tr><td style="width: 25%;"><a href="modifylist.html">modifylist</a></td>
<td>Modification of an ACNUC list</td></tr>
<tr><td style="width: 25%;"><a href="move.html">move</a></td>
<td>Rename an R object</td></tr>
<tr><td style="width: 25%;"><a href="msf.html">msf</a></td>
<td>Example of results obtained after a call to read.alignment</td></tr>
<tr><td style="width: 25%;"><a href="move.html">mv</a></td>
<td>Rename an R object</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="n2s.html">n2s</a></td>
<td>function to convert the numeric encoding of a DNA sequence into a vector of characters</td></tr>
<tr><td style="width: 25%;"><a href="get.ncbi.html">ncbi.fna.url</a></td>
<td>Bacterial complete genome data from ncbi ftp site</td></tr>
<tr><td style="width: 25%;"><a href="get.ncbi.html">ncbi.gbk.url</a></td>
<td>Bacterial complete genome data from ncbi ftp site</td></tr>
<tr><td style="width: 25%;"><a href="get.ncbi.html">ncbi.ptt.url</a></td>
<td>Bacterial complete genome data from ncbi ftp site</td></tr>
<tr><td style="width: 25%;"><a href="get.ncbi.html">ncbi.stats</a></td>
<td>Bacterial complete genome data from ncbi ftp site</td></tr>
</table>

<h2><a name="O">-- O --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="oriloc.html">oriloc</a></td>
<td>Prediction of origin and terminus of replication in bacteria.</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="parser.socket.html">parser.socket</a></td>
<td>Utility function to parse answers from an ACNUC server</td></tr>
<tr><td style="width: 25%;"><a href="peakabif.html">peakabif</a></td>
<td>Extraction of Peak locations, Heights and Surfaces from ABIF data</td></tr>
<tr><td style="width: 25%;"><a href="permutation.html">permutation</a></td>
<td>Sequence permutation according to several different models</td></tr>
<tr><td style="width: 25%;"><a href="prepgatannots.html">pga</a></td>
<td>Select annotation lines in an ACNUC database</td></tr>
<tr><td style="width: 25%;"><a href="phylip.html">phylip</a></td>
<td>Example of results obtained after a call to read.alignment</td></tr>
<tr><td style="width: 25%;"><a href="pK.html">pK</a></td>
<td>pK values for the side chain of charged amino acids from various sources</td></tr>
<tr><td style="width: 25%;"><a href="plot.SeqAcnucWeb.html">plot.SeqAcnucWeb</a></td>
<td>To Plot Subsequences on the Parent Sequence</td></tr>
<tr><td style="width: 25%;"><a href="plotabif.html">plotabif</a></td>
<td>Electrophoregram plot for ABIF data</td></tr>
<tr><td style="width: 25%;"><a href="plotladder.html">plotladder</a></td>
<td>Simple plot of an allelic ladder from ABIF data</td></tr>
<tr><td style="width: 25%;"><a href="plotPanels.html">plotPanels</a></td>
<td>Representation of Amplicon Size Ranges of a STR kit.</td></tr>
<tr><td style="width: 25%;"><a href="pmw.html">pmw</a></td>
<td>Protein Molecular Weight</td></tr>
<tr><td style="width: 25%;"><a href="prepgatannots.html">prepgetannots</a></td>
<td>Select annotation lines in an ACNUC database</td></tr>
<tr><td style="width: 25%;"><a href="prettyseq.html">prettyseq</a></td>
<td>Text representation of a sequence from an ACNUC server</td></tr>
<tr><td style="width: 25%;"><a href="print.qaw.html">print.qaw</a></td>
<td>Print method for objects from class qaw</td></tr>
<tr><td style="width: 25%;"><a href="print.SeqAcnucWeb.html">print.SeqAcnucWeb</a></td>
<td>Print method for objects from class SeqAcnucWeb</td></tr>
<tr><td style="width: 25%;"><a href="prochlo.html">prochlo</a></td>
<td>Zscore on three strains of Prochlorococcus marinus</td></tr>
</table>

<h2><a name="Q">-- Q --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="query.html">query</a></td>
<td>To get a list of sequence names from an ACNUC data base located on the web</td></tr>
<tr><td style="width: 25%;"><a href="acnucopen.html">quitacnuc</a></td>
<td>open and close a remote access to an ACNUC database</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="read.abif.html">read.abif</a></td>
<td>Read ABIF formatted files</td></tr>
<tr><td style="width: 25%;"><a href="read.alignment.html">read.alignment</a></td>
<td>Read aligned sequence files in mase, clustal, phylip, fasta or msf format</td></tr>
<tr><td style="width: 25%;"><a href="read.fasta.html">read.fasta</a></td>
<td>read FASTA formatted files</td></tr>
<tr><td style="width: 25%;"><a href="getAnnot.html">readAnnots.socket</a></td>
<td>Generic Function to get sequence annotations</td></tr>
<tr><td style="width: 25%;"><a href="readBins.html">readBins</a></td>
<td>Import GenMapper Bins configuration file</td></tr>
<tr><td style="width: 25%;"><a href="read.fasta.html">readfasta</a></td>
<td>read FASTA formatted files</td></tr>
<tr><td style="width: 25%;"><a href="readfirstrec.html">readfirstrec</a></td>
<td>Low level function to get the record count of the specified ACNUC index file</td></tr>
<tr><td style="width: 25%;"><a href="readPanels.html">readPanels</a></td>
<td>Import GenMapper Panels configuration file</td></tr>
<tr><td style="width: 25%;"><a href="readsmj.html">readsmj</a></td>
<td>Low level function to read ACNUC SMJYT index files</td></tr>
<tr><td style="width: 25%;"><a href="rearranged.oriloc.html">rearranged.oriloc</a></td>
<td>Detection of replication-associated effects on base composition asymmetry in prokaryotic chromosomes.</td></tr>
<tr><td style="width: 25%;"><a href="recstat.html">recstat</a></td>
<td>Prediction of Coding DNA Sequences.</td></tr>
<tr><td style="width: 25%;"><a href="residuecount.html">residuecount</a></td>
<td>Total number of residues in an ACNUC list</td></tr>
<tr><td style="width: 25%;"><a href="revaligntest.html">revaligntest</a></td>
<td>Three aligned nucleic acid sequences</td></tr>
<tr><td style="width: 25%;"><a href="reverse.align.html">reverse.align</a></td>
<td>Reverse alignment - from protein sequence alignment to nucleic sequence alignment</td></tr>
<tr><td style="width: 25%;"><a href="zscore.html">rho</a></td>
<td>Statistical over- and under- representation of dinucleotides in a sequence</td></tr>
<tr><td style="width: 25%;"><a href="rot13.html">rot13</a></td>
<td>Ergheaf gur EBG-13 pvcurevat bs n fgevat</td></tr>
<tr><td style="width: 25%;"><a href="uco.html">rscu</a></td>
<td>Codon usage indices</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="s2c.html">s2c</a></td>
<td>conversion of a string into a vector of chars</td></tr>
<tr><td style="width: 25%;"><a href="s2n.html">s2n</a></td>
<td>simple numerical encoding of a DNA sequence.</td></tr>
<tr><td style="width: 25%;"><a href="savelist.html">savelist</a></td>
<td>Save sequence names or accession numbers into a file</td></tr>
<tr><td style="width: 25%;"><a href="SeqAcnucWeb.html">SeqAcnucWeb</a></td>
<td>Sequence coming from a remote ACNUC data base</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastaAA.html">SeqFastaAA</a></td>
<td>AA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastadna.html">SeqFastadna</a></td>
<td>Class for DNA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="SeqFrag.html">SeqFrag</a></td>
<td>Class for sub-sequences</td></tr>
<tr><td style="width: 25%;"><a href="seqinr-package.html">seqinr</a></td>
<td>Biological Sequences Retrieval and Analysis</td></tr>
<tr><td style="width: 25%;"><a href="SEQINR.UTIL.html">SEQINR.UTIL</a></td>
<td>utility data for seqinr</td></tr>
<tr><td style="width: 25%;"><a href="setlistname.html">setlistname</a></td>
<td>Sets the name of an ACNUC list identified by its rank</td></tr>
<tr><td style="width: 25%;"><a href="splitseq.html">splitseq</a></td>
<td>split a sequence into sub-sequences</td></tr>
<tr><td style="width: 25%;"><a href="stresc.html">stresc</a></td>
<td>Utility function to escape LaTeX special characters present in a string</td></tr>
<tr><td style="width: 25%;"><a href="stutterabif.html">stutterabif</a></td>
<td>Stutter ratio estimation</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastaAA.html">summary.SeqFastaAA</a></td>
<td>AA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="SeqFastadna.html">summary.SeqFastadna</a></td>
<td>Class for DNA sequence in Fasta Format</td></tr>
<tr><td style="width: 25%;"><a href="swap.html">swap</a></td>
<td>Exchange two R objects</td></tr>
<tr><td style="width: 25%;"><a href="syncodons.html">syncodons</a></td>
<td>Synonymous codons</td></tr>
<tr><td style="width: 25%;"><a href="synsequence.html">synsequence</a></td>
<td>Random synonymous coding sequence generation</td></tr>
</table>

<h2><a name="T">-- T --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="tablecode.html">tablecode</a></td>
<td>to plot genetic code as in textbooks</td></tr>
<tr><td style="width: 25%;"><a href="test.co.recstat.html">test.co.recstat</a></td>
<td>Tests if regions located between Stop codons contain putative CDSs.</td></tr>
<tr><td style="width: 25%;"><a href="test.li.recstat.html">test.li.recstat</a></td>
<td>Tests if regions located between Stop codons contain putative CDSs.</td></tr>
<tr><td style="width: 25%;"><a href="toyaa.html">toyaa</a></td>
<td>A toy example of amino-acid counts in three proteins</td></tr>
<tr><td style="width: 25%;"><a href="toycodon.html">toycodon</a></td>
<td>A toy example of codon counts in three coding sequences</td></tr>
<tr><td style="width: 25%;"><a href="translate.html">translate</a></td>
<td>Translate nucleic acid sequences into proteins</td></tr>
<tr><td style="width: 25%;"><a href="trimSpace.html">trimSpace</a></td>
<td>Trim leading and/or trailing spaces in strings</td></tr>
</table>

<h2><a name="U">-- U --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="uco.html">uco</a></td>
<td>Codon usage indices</td></tr>
<tr><td style="width: 25%;"><a href="ucoweight.html">ucoweight</a></td>
<td>Weight of each synonymous codon</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="waterabs.html">waterabs</a></td>
<td>Light absorption by the water column</td></tr>
<tr><td style="width: 25%;"><a href="where.is.this.acc.html">where.is.this.acc</a></td>
<td>Scans databases for a given sequence accession number</td></tr>
<tr><td style="width: 25%;"><a href="words.html">words</a></td>
<td>To get all words from an alphabet.</td></tr>
<tr><td style="width: 25%;"><a href="words.pos.html">words.pos</a></td>
<td>Positions of possibly degenerated motifs within sequences</td></tr>
<tr><td style="width: 25%;"><a href="write.fasta.html">write.fasta</a></td>
<td>Write sequence(s) into a file in fasta format</td></tr>
</table>

<h2><a name="Z">-- Z --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="zscore.html">zscore</a></td>
<td>Statistical over- and under- representation of dinucleotides in a sequence</td></tr>
</table>
</body></html>
